Ifeanyi/Chapter_2/exercise_2.14.5

```diff
@@ -0,0 +1,8 @@
+#When interest rate is 1%
+P = input('what is Principal amount: ')
```

Comment:
You've just repeated the variable names as given in the question. It's not human readable in any way!